frontend/src/components/design/StyledHowToStart.tsx

```diff
@@ -0,0 +1,14 @@
+import { Box, Button } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+export const StyledMapWrapper = styled(Box)`
+  height: 100%;
+  width: 100%;
+`;
+export const Text = styled(Box)`
+  margin: 20px 0 0 0;
+`;
+export const ButtonTitle = styled(Button)`
+  text-transform: none;
+  font-size: 16px;
+`;
```

Comment:
![Screenshot 2022-07-12 at 12-55-55 MapUa App](https://user-images.githubusercontent.com/88610270/178464201-6382afb4-19ef-4b7d-a90f-54bb8278e4c6.png)
Add font-weigth:400 so instruction button will have the same style as other links
![3rd](https://user-images.githubusercontent.com/88610270/178464570-068e8ae2-4fc1-45aa-9293-45e4d1cc93e0.png)
![Screenshot 2022-07-12 at 12-58-42 MapUa App](https://user-images.githubusercontent.com/88610270/178464702-7f5a6ca3-9b23-499a-9def-cdb09d0a8822.png)

Reply:
added